README.md

```diff
@@ -27,115 +31,164 @@ repositories' pull requests by using the GitHub Pull Request Builder plugin. It
+of Python code, yamllint validation of all package metadata YAML files and rpmlint
+validation of all RPM packages specification files. These tests can be triggered
+independently with the phrases "start build", "start pylint", "start yamllint"
+and "start rpmlint" respectively.
+
+### Periodic builds process
+
+A Jenkins job will trigger periodically to check the versions of the packages
+available in OpenPOWER Host OS GitHub organization, update the corresponding
+packages' versions in the "versions" git repository, create a build with all
+packages available and create release notes, by executing the corresponding
+commands from the build scripts in the "builds" git repository. Commits will
+be created for the local "versions" and "open-power-host-os.github.io" git
+repositories and pushed to the designated GitHub user's git repository. Ideally,
+this should be a "bot" user. Pull requests for those commits can be created
+manually and merged into the main organization/user git repository, if desired.
+
+Note: The default is to execute those periodic builds once a week (every Wednesday
+at 11 AM UTC-03:00).
 
 ## Infrastructure setup
 
 If the whole infrastructure setup is desired, you should do the following steps:
-- Prepare CentOS system
-- Execute Ansible playbooks
-- Manual Jenkins steps
+- Prepare CentOS system(s)
+- Setup SSH credentials used to access upload server and GitHub from Jenkins slaves
+- Automatically setup Jenkins master and slave(s) using Ansible playbooks
+- Manually setup Jenkins via web UI
  - Create administrative jobs
- - Create slaves credentials
- - Create GitHub credentials
- - Create slave nodes
- - Create Host OS builds jobs
+ - Setup SSH credentials used to access Jenkins slaves from Jenkins master
+ - Create credentials used to access GitHub API from Jenkins
+ - Create Jenkins slaves in Jenkins web UI
+ - Create builds jobs
+
 They are described in detail below.
 
-### Prepare CentOS system
+### Prepare CentOS system(s)
+
+First, you will need to prepare at least one CentOS system. If the Jenkins
+instance will be used for testing, one system is enough; production instances
+should use more systems.  Currently, OpenPOWER Host OS has 1 Jenkins slave for
+validating the code from GitHub pull requests and 3 Jenkins slaves for builds.
+
+KVM virtual machines are recommended, but you could optionally use bare-metal or
+containers instead. If KVM virtualization is used, for each required system,
+you will need to create a KVM guest and install a CentOS ISO in it. It is
+recommended to use virt-install to do it.
+
+If you do not have a direct network connection to the system(s), you may
+setup a SSH proxy in ~/.ssh/config in your workstation to make the SSH
+proxy transparent. Example:
 
-First, you'll need to prepare at least one CentOS system. Virtual machines are
-recommended, but they could also be bare-metal or containers.
+Host ophostos-builds
```

Comment:
I'd write `open-power-host-os-builds` to be consistent with the name in other places.